app/assets/javascripts/controllers/container_dashboard/container_dashboard_controller.js

```diff
@@ -0,0 +1,71 @@
+angular.module('containerDashboard', ['ui.bootstrap', 'patternfly', 'patternfly.charts'])
+    .config(['$httpProvider', function($httpProvider) {
+        $httpProvider.defaults.headers.common['X-CSRF-Token'] = jQuery('meta[name=csrf-token]').attr('content');
+    }])
+    .controller('containerDashboardController', ['$scope', 'ChartsDataMixin', '$http', '$interval', "$location",
+        function($scope, chartsDataMixin, $http, $interval, $location) {
+            dashboard = this;
+            dashboard.utilizationLoadingDone = false; // Is this 'loading done' mechanism needed?
```

Comment:
@abonas So all the data is retrieved simultaneously. Is there a point in showing the loading icon on all the cards?
if there is, is there value in having all these ```loadingDone``` variables or having just one for all cards?